### DISCUSSION TITLE
Appraise Notifications using Outlook Email Address

### DISCUSSION THREAD

**Discussion by trophyguy** [mcguirepr89/BirdNET-Pi#630] [Q&A]
I'm giving this a shot now. From searching through the repo, there seems to be a number of different configurations for Appraise & an Outlook.com email address. Here's what I have tried so far:

mailtos://?smtp=smtp.outlook.com&from=tony.taurisano@outlook.com&pass={my.password.here}
mailtos://?smtp=smtp-mail.outlook.com&from=tony.taurisano@outlook.com&pass={my.password.here}
mailto://tony.taurisano:{my.password.here}@outlook.com

When I test the notifications I get:
Unparseable E-Mail URL mailtos://?smtp=smtp.outlook.com&from=tony.taurisano@outlook.com&pass={my.password.here}
Unparseable E-Mail URL mailtos://?smtp=smtp-mail.outlook.com&from=tony.taurisano@outlook.com&pass={my.password.here}
INFO - Applying Custom Defaults
INFO - Notifying 1 service(s) asynchronously.
WARNING - A Connection error occurred sending Email notification to outlook.com.

I have 2FA applied to my Outlook account. Would that be an issue for trying to get these to work?
Thanks!

**Comment by Vertikar** [comment]
2FA will definitely be causing issues, you'll need to use an app password instead:
https://support.microsoft.com/en-us/account-billing/using-app-passwords-with-apps-that-don-t-support-two-step-verification-5896ed9b-4263-e681-128a-a6f2979a7944

**Reply by trophyguy** [reply]
I tried what you suggested and got the same errors as I posted above using an App Password.

**Comment by Vertikar** [comment]
Ah, not sure then, might need one of the Devs or someone more knowledgeable
than me to help you grab some logs and investigate further.

**Comment by ehpersonal38** [comment]
To start, the formatting of your first 2 config lines are incorrect. They should be like:
`mailtos://outlook.com?smtp=smtp-mail.outlook.com&from=USERNAME@outlook.com&user=USERNAME&pass=PASSWORD`

Your third line is fine though and should be working, but it's not for some reason. I actually tried all 3 of your configs (just formatted properly) with my own Outlook account and couldn't get any of them to work. So I think the next place to ask for assistance will be with the folks on the Apprise GitHub repo. Hopefully you can get to the bottom of this one, I wish you the best of luck!

---

BTW if you want to continue testing, I would use the following steps instead of the "Send Test Notification" button (it has a bug where the `&` symbol will break the formatting):

1) Put your Apprise config in Tools -> Settings like you normally would, and save. 
2) From now on whenever you want to send a test notification, run the following command in your terminal:
`$HOME/BirdNET-Pi/birdnet/bin/apprise -vv -t 'TEST' -b "TEST" --config=$HOME/BirdNET-Pi/apprise.txt`

**Comment by PaPaTheGMan** [comment]
I know this is old, but just wanted to add, for the record, that I did get a variation of #1 to work.  Remove the ' and replace what s inside with your particulars.
mailtos://'Insert USER'@outlook.com?smtp=smtp-mail.outlook.com&from='Insert desired from'@outlook.com&user='outlook user name'@outlook.com&pass='two factor app password'
